packages/kitsu-core/src/serialise/index.spec.js

```diff
@@ -13,7 +13,7 @@ const serial = {
   // camelCaseTypes: false  resourceCase: none    pluralize: false
   none: serialise.bind({ camel: skip, resCase: skip, plural: skip }),
   // camelCaseTypes: false  resourceCase: none    pluralize: true
-  nonePlural: serialise.bind({ camel: skip, resCase: skip, plural }),
+  nonePlural: serialise.bind({ camel: skip, resCase: skip, plural: skip }),
```

Comment:
Please don't change this - use `none` for the test. These parameters in the serial object are just listing all the possible values the 3 options can have.